code/client.dm

```diff
@@ -1277,6 +1285,15 @@ var/global/curr_day = null
 
 	apply_depth_filter() //see _plane.dm
 
+/client/verb/apply_parallax()
+	set hidden = 1
+	set name ="apply-parallax"
+
+	parallax = !parallax // fps impact, unknown as to how much
+
+	if(mob?.parallax)
```

Comment:
why do you think this will be null

Reply:
i completely didnt realize that i didnt need to use .? there